noir-projects/noir-contracts/contracts/avm_test_contract/Nargo.toml

```
@@ -7,4 +7,6 @@ type = "contract"
 [dependencies]
 aztec = { path = "../../../aztec-nr/aztec" }
 compressed_string = { path = "../../../aztec-nr/compressed-string" }
-sha256 = { tag = "v0.1.0", git = "https://github.com/noir-lang/sha256" }
\ No newline at end of file
+sha256 = { tag = "v0.1.0", git = "https://github.com/noir-lang/sha256" }
```

```suggestion
sha256 = { tag = "v0.1.2", git = "https://github.com/noir-lang/sha256" }
```
May conflict with https://github.com/AztecProtocol/aztec-packages/pull/11696